kotlin/workflow-ui/modal-android/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Square Inc.
```

Comment:
Super nit: `2017` -> `2020` ? 😄

Reply:
Some of these are for code that was written earlier.